revocation/result/results.go

```diff
@@ -100,4 +168,7 @@ type CertRevocationResult struct {
 	// Otherwise, every server specified had some error that prevented the
 	// status from being retrieved. These are all contained here for evaluation
 	ServerResults []*ServerResult
```

Comment:
I think it had better to be another PR because we will refactor the OCSP package.
Added an issue https://github.com/notaryproject/notation-core-go/issues/222

Reply:
I'd say we should do it in this iteration, because it's directly related to the `Result`, i.e. it also impacts users of CRL. It should be a quick fix though?

Reply:
Tried to refactor, but it seems redundant to always copy the field from `OCSPResults` to `ServerResults`, which reduces readability. I think we should track this as an issue and rename the field in core-go v2.